themes/default/content/docs/guides/crosswalk/aws/api-gateway.md

```diff
@@ -22,23 +22,26 @@ is inexpensive, has no minimum fees, and you only pay for the API calls you rece
 
 ## Overview
 
-Pulumi Crosswalk for Amazon Web Services (AWS) provides better AWS API management through significantly easier ways of programming an API Gateway. This includes using
-infrastructure as code techniques for simple, declarative APIs, including easy Lambda-based handlers.
+Pulumi Crosswalk for Amazon Web Services (AWS) provides better AWS API management through significantly easier
+ways of programming an API Gateway. This includes using infrastructure as code techniques for simple, declarative
+APIs, including easy Lambda integration.
 
-## Defining an AWS API Gateway Endpoint and Routes
+Full examples can be found in the [Pulumi Registry for the AWS API Gateway component](https://www.pulumi.com/registry/packages/aws-apigateway/).
+
+## Create and Configure Routes
 
 AWS API Gateway creates REST APIs that:
 
-* Are HTTP based.
-* Adhere to the [REST](https://en.wikipedia.org/wiki/Representational_state_transfer) protocol.
-* Implement standard HTTP methods such as `GET`, `POST`, `PUT`, `PATCH`, and `DELETE`.
+- Are HTTP based.
+- Adhere to the [REST](https://en.wikipedia.org/wiki/Representational_state_transfer) protocol.
+- Implement standard HTTP methods such as `GET`, `POST`, `PUT`, `PATCH`, and `DELETE`.
 
 Each API Gateway instance defines a new API endpoint and a collection of API routes, each of which has a distinct URL.
 
 > Internally the API Gateway resource uses a collection of supporting objects, like resources, methods, and more,
 > however one of the benefits of Pulumi Crosswalk for AWS is that it hides these mechanics behind a simpler interface.
 
```

Comment:
this isnt you, but would you mind change this to the note styles. we shouldn't be using quote styles for things that are not quotes.

```
{{% notes type="info" %}}

{{% /notes %}}
```